aqi_monitor/requirements.txt

```diff
@@ -1 +1,10 @@
-Django==3.0.7
\ No newline at end of file
+amqp==2.6.0
```

Comment:
Where did you get these dependencies? I guess django and celery would be enough

Reply:
Fixed